test/web-platform-tests/to-run.yaml

```diff
@@ -33,6 +33,7 @@ cors-safelisted-request-header.any.html: [fail, Depends on fetch]
 image-tainting-in-cross-origin-iframe.sub.html: [timeout, Unknown]
 origin.htm: [timeout, https://github.com/jsdom/jsdom/issues/1833]
 preflight-cache.htm: [timeout, Cache should probably be implemented for simple requests before]
+preflight-failure.htm: [timeout, Node throws an error on code 304, done() is called multiple times]
```

Comment:
This should probably support a value like: `timeout‑node15` or something.

Reply:
Would that make the test run for Node v14 and older?

Reply:
I think that's the implication. IMO it's fine as-is though. Getting improved test coverage on older node versions is not worth the burden I think.